src/NetAnalyzers/Core/Microsoft.NetCore.Analyzers/MicrosoftNetCoreAnalyzersResources.resx

```diff
@@ -1519,4 +1519,13 @@
     <value>and all other platforms</value>
     <comment>This call site is reachable on: 'windows' 10.0.2000 and later, and all other platforms</comment>
   </data>
+  <data name="DetectPreviewFeaturesDescription" xml:space="preserve">
+    <value>An assembly has to opt into preview features before using them.</value>
+  </data>
+  <data name="DetectPreviewFeaturesMessage" xml:space="preserve">
+    <value>Enable preview features for this assembly</value>
```

Comment:
This needs a much more informative message, similar to the descriptive messages that @buyaa-n has for platform compat analyzer diagnostics.

Reply:
Yup yup, the message/title/description will change in the future. We need to work with the docs team to come up with a resource that explains how preview features work. Then we can update the message here to point to that resource. At the moment, I want to get a working implementation up so we can iterate on the final shape for `[RequiresPreviewFeatures]` better.

Reply:
For the time-being, while we prepare to start demoing the feature, would the following message be easy to get in place?

`Using {api} requires opting into preview features. See https://aka.ms/dotnet-warnings/preview-features for more information.`